tests/sentry/manager/test_team_manager.py

```diff
@@ -1,13 +1,17 @@
-from sentry.models import Team, User
+from sentry.app import env
+from sentry.models import Team
 from sentry.services.hybrid_cloud.user.service import user_service
 from sentry.testutils.cases import TestCase
 from sentry.testutils.silo import region_silo_test
 
 
 @region_silo_test
 class TeamManagerTest(TestCase):
+    def setUp(self) -> None:
+        env.clear()
```

Comment:
Should you call `super.setUp()` here?

Reply:
I believe it's a no-op but, yeah, I'll put that in for hygiene's sake.